docs/help.md

```diff
@@ -0,0 +1,27 @@
+### Commands
+`bot adapter` - Reply with the adapter
+
+`bot create|close milestone all "title"` - Create or close all milestones listed in `config.json`
```

Comment:
think it not creates all milestones listed in `config.json`, but creates milestone for **all repos** listed in `config.json`

Reply:
will be fixed